src/tirith/providers/json/handler.py

```diff
@@ -43,7 +43,12 @@ def get_value(provider_args: Dict, input_data: Dict) -> List[dict]:
             )
         ]
 
-    outputs = [create_result_dict(value=value, meta=None, err=None) for value in values]
+    # Create result dict with addresses as a separate property, not in meta
+    outputs = []
+    for value in values:
+        result = create_result_dict(value=value, meta=None, err=None)
+        result["addresses"] = key_path
```

Comment:
Hi Akash, thanks, could you also make sure that every `result["addresses"]` is always a list of strings. For example:
```suggestion
        result["addresses"] = [key_path]
```